samples/security/vulnerability findings/slack_notifications_for_critical_vulnerabilities.yaml

```diff
@@ -0,0 +1,232 @@
+          				"action_id": "button-action"
+          			}
+          		}
+          	]
+          }
+        reaction: []
+        connection: ""
+        workflowID: "{{ execution().workflow.id }}"
+        channelType: id
+        executionID: "{{ execution().id }}"
+        executionDate: "{{ execution().started_at }}"
+        appendToThread: false
+        selectedRequestType: 0
+        attachmentToggleValue: none
+      position:
+        x: 0
+        y: 2
+      predecessors:
+        - get_vulnerabilites
+      conditions:
+        states:
+          get_vulnerabilites: SUCCESS
+        custom: '{{result("get_vulnerabilites").records|length > 0}}'
+      concurrency: 1
+      withItems: finding in {{ result("get_vulnerabilites").records }}
+    get_vulnerabilites:
```

Comment:
@christophpre-dynatrace I tried to execute the workflow in umsay, the DQL stopped with an error after 5:55h. Do you see any change to optimize the query or limit it to maybe 2h, or provide some hints around scope and execution time?